exonum-java-binding-core/src/test/java/com/exonum/binding/service/adapters/UserServiceAdapterTest.java

```diff
@@ -73,58 +73,45 @@
   private UserServiceAdapter serviceAdapter;
 
   private static final short SERVICE_ID = (short) 0xA103;
+  private static final short TRANSACTION_ID = 1;
   private static final long SNAPSHOT_HANDLE = 0x0A;
   private static final long HEIGHT = 1;
   private static final int VALIDATOR_ID = 1;
 
   @Test
   void convertTransaction_ThrowsIfNull() {
-    assertThrows(NullPointerException.class, () -> serviceAdapter.convertTransaction(null));
+    assertThrows(NullPointerException.class, () ->
+        serviceAdapter.convertTransaction(SERVICE_ID, TRANSACTION_ID, null));
   }
 
   @Test
   void convertTransaction() {
     Transaction expectedTransaction = mock(Transaction.class);
-    when(service.getId()).thenReturn(SERVICE_ID);
-    when(service.convertToTransaction(any(BinaryMessage.class)))
+    when(service.convertToTransaction(any(RawTransaction.class)))
         .thenReturn(expectedTransaction);
 
-    byte[] message = getServiceMessage(SERVICE_ID)
-        .getSignedMessage()
-        .array();
+    byte[] message = TemplateRawTransaction.createRawTransaction(SERVICE_ID).getPayload();
```

Comment:
This is *not* a message. Also, why do we perform such complex expression to just create an array which contents is not relevant? I'd replace with `new byte[1]`